app/src/main/java/com/jerboa/ui/components/post/PostListing.kt

```diff
@@ -1645,9 +1644,8 @@ fun PostListingCard(
                 .padding(vertical = MEDIUM_PADDING)
                 .clickable { onPostClick(postView) }
                 .testTag("jerboa:post"),
-        verticalArrangement = Arrangement.spacedBy(MEDIUM_PADDING),
```

Comment:
Why remove this one, and add the spacers?

Reply:
See StackOverflow link

Reply:
Ah I see, that's a weird one.